doc/contributing/collaborator-guide.md

```diff
@@ -392,13 +392,21 @@ metadata. Raise a pull request like any other change.
 
 Treat commits that introduce new core modules with extra care.
 
-Check if the module's name conflicts with an existing ecosystem module. If it
-does, choose a different name unless the module owner has agreed in writing to
-transfer it.
+New modules must be added with the `node:` prefix.
 
-If the new module name is free, register a placeholder in the module registry as
-soon as possible. Link to the pull request that introduces the new core module
-in the placeholder's `README`.
+When adding promises to an existing API, add `/promises`
+(`inspector/promises`, etc.). Apply the `semver-major` label to the addition.
+
+If the new module name without the `node:` prefix is free, register
```

Comment:
I don't think we need to call out `node:` here since it is never available on npm. What about this?

```
If the new module name is available in the module registry, reserve it as soon as possible.
```